packages/visual-sdk/package.json

```diff
@@ -44,7 +44,7 @@
     "tsc": "tsc --noEmit",
     "test": "jest --watch",
     "test:ci": "jest",
-    "coverage": "vitest --coverage",
+    "coverage": "jest --coverage",
```

Comment:
why are we using jest again instead of vitest? :)

Reply:
the visual sdk uses swc for the build and not vite, and therefore it's easier to setup jest with swc :) 
otherwise we would need to have still the vite dependency, add having a vite config